src/examples/table/TableVerticalScrolling.tsx

```diff
@@ -18,15 +18,15 @@ const rowData = Array.from(Array(10)).map((row, index) => ({
   soil: 'Well draining'
 }));
 
-const StyledHeaderCell = styled(HeaderCell)`
+const StyledHeaderCell = styled(Table.HeaderCell)`
   box-shadow: inset 0 -1px 0 ${props => getColorV8('neutralHue', 300, props.theme)};
```

Comment:
Shoudl this swap over to `getColor`? Not sure if there's other custom colors in here. Also not sure if this can take advantage of the new `shadow` variables. 😮